cobra2snooty_test.go

```diff
@@ -57,6 +57,10 @@ func Echo() *cobra.Command {
 		Short:   "Echo anything to the screen",
 		Long:    "an utterly useless command for testing",
 		Example: "Just run root echo",
+		Annotations: map[string]string{
+			"args":                "string to print",
+			"string to printDesc": "A string to print",
+		},
```

Comment:
Should the test include multiple and required args?
```suggestion
		Annotations: map[string]string{
			"args":                "string to print, second string",
			"requiredArgs":  "string to print, second string",
			"string to printDesc": "A string to print",
			"second stringDesc": "A second string",
		},
```

Reply:
wanted to keep it simple but why not